<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>create_course</class>
 <widget class="QDialog" name="create_course">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Instructors Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_load">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>490</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>load</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_create">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>510</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>create</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="date_startDate">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>End Date</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Text_instructorEmployeeID">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>180</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="date_endDate">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Text_courseName">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Couse Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>341</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>* to assign TA's click select the TA ID row that identifies the Ta
 *for multiple TA's click TA ID rows while holding the key Ctrl</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>170</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Instrucor
EmployeerID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Date</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="Ta_list">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>280</y>
     <width>211</width>
     <height>192</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <column>
    <property name="text">
     <string>TA Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>TA ID</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="Text_instructorName">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
